### ISSUE TITLE
ids v1.0.4 build error

### ISSUE THREAD

**Issue by june2315** [bpmn-io/ids#8]
### Describe the Bug

ids v1.0.4 build error 

I delete my project node_modules folder and run yarn install in Terminal, then run yarn build, but build failed.

### Steps to Reproduce

1. delete node_modules folder
2. yarn install 
3. yarn build
4. now this happens

Module not found: Error: Can't resolve 'ids' in '/xxx/node_modules/bpmn-js/lib/draw'
 @ ./node_modules/bpmn-js/lib/draw/index.js 1:0-42 8:26-38
 @ ./node_modules/bpmn-js/lib/core/index.js 1:0-33 6:4-14
 @ ./node_modules/bpmn-js/lib/Viewer.js 3:0-32 66:2-12
 @ ./node_modules/bpmn-js/lib/NavigatedViewer.js 3:0-30 16:2-13 19:26-32 29:2-27

### Expected Behavior

build success

### Environment

- Host node.js v16.20.0, yarn v1.22.19
- OS: macOS 13.4.1
- Library version: ["bpmn-js": "^8.8.3", "ids": "1.0.4"]

**Comment by loriling** [comment]
I meet the same issue..

**Comment by ialiaslani** [comment]
I meet the same issue too dist is not on npm

**Comment by itfiwn4653** [comment]
I meet the same issue..
not only build meet the same issue
after npm install, i cannot start dev server
 

```
...
{
          location: {
            column: 16,
            file: 'node_modules/_bpmn-js@9.3.1@bpmn-js/lib/draw/BpmnRenderer.js',
            length: 5,
            line: 58,
            lineText: "import Ids from 'ids';",
            namespace: '',
            suggestion: ''
          },
          text: 'The plugin "vite:dep-pre-bundle" was triggered by this import'
        }
      ],
      pluginName: 'vite:dep-pre-bundle',
      text: 'Failed to resolve entry for package "ids". The package may have incorrect main/module/exports specified in its package.json.'
```

**Comment by itfiwn4653** [comment]
Now the easiest solution is 
after npm install 

1. cd into the bpmn.js folder
2. change package.json 

`"ids": "1.0.3",`

3. npm install --production

now everything is ok

**Comment by RazorYui** [comment]
Its because node_modules/ids missed dist folder in. v1.0.4
My solution was install older version of package with npm i ids@1.0.3 in project directory

Fix it, please

**Comment by webxmsj** [comment]
I meet the same issue..

**Comment by turenc** [comment]
Do you have an explanation of the criteria for your code deletion?

**Comment by keepwe** [comment]
I meet the same issue..

**Comment by keepwe** [comment]
临时解决方案：
在用到bpmn.js依赖的项目中，使用resolutions修改间接依赖/深层依赖。
具体实现：在package.json中与dependencies和devDependencies同级加入  "resolutions": {  "ids": "1.0.0" },【yarn才会生效，若使用npm安装的还需要在scripts中加入 "preinstall": "npx force-resolutions",】

**Comment by nikku** [comment]
It works:

```sh
$ npm install ids@1.x

added 1 package, changed 7 packages, and audited 1337 packages in 5s
```

```sh
$ npm ls ids
test-project@0.0.0 {...}/test-project
└── ids@1.0.4
```

**Comment by nikku** [comment]
Released v1.0.5 with `dist` folder.
